pom.xml

```diff
@@ -117,6 +117,10 @@
 			<url>https://hub.spigotmc.org/nexus/content/repositories/snapshots</url>
 		</repository>
 		<repository>
+			<id>dmulloy2-repo</id>
```

Comment:
I'll upload Spigot artifacts on my own Maven repo, this way we can get rid of both `spigotmc.org` and `dmulloy2.net` repositories. I'll tell you when it's done.